### ISSUE TITLE
Error on Linux when running policy tests

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#2365]
Stacktrace:

```
File "/usr/lib/python2.7/unittest/case.py", line 329, in run
    testMethod()
  File "/home/jenkins/workspace/GH_Tribler_PR_tests_linux/tribler/Tribler/Test/test_as_server.py", line 81, in check
    result = fun(*argv, **kwargs)
  File "/home/jenkins/workspace/GH_Tribler_PR_tests_linux/tribler/Tribler/Test/Core/CreditMining/test_creditmining.py", line 52, in test_random_policy
    self.assertEqual(3, len(ids_start), "Start failed %s vs %s" % (ids_start, torrents_start))
  File "/usr/lib/python2.7/unittest/case.py", line 513, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/usr/lib/python2.7/unittest/case.py", line 506, in _baseAssertEqual
    raise self.failureException(msg)
"Start failed [4, 2] vs [{'num_seeders': 4, 'creation_date': 4, 'num_leechers': 3, 'metainfo': <Tribler.Test.Core.CreditMining.mock_creditmining.MockMeta object at 0x7fb437f3bf50>}, {'num_seeders': 2, 'creation_date': 2, 'num_leechers': 1, 'metainfo': <Tribler.Test.Core.CreditMining.mock_creditmining.MockMeta object at 0x7fb437f3bf10>}]
```

Reference: https://jenkins.tribler.org/job/GH_Tribler_PR_tests_linux/2281/testReport/junit/Tribler.Test.Core.CreditMining.test_creditmining/TestBoostingManagerPolicies/test_random_policy/

**Comment by ardhipoetra** [comment]
Question, how do I test 'randomness'?

**Comment by lfdversluis** [comment]
@ardhipoetra By running it a few times until it crashes and investigate

**Comment by ardhipoetra** [comment]
This one is solved in PR #2461. Close it, @devos50?

**Comment by devos50** [comment]
@ardhipoetra great, I will close it 👍
